samples/android/app/src/main/java/org/hyperledger/indy/sdk/utils/PoolJson.java

```diff
@@ -0,0 +1,9895 @@
+package org.hyperledger.indy.sdk.utils;
+
+public class PoolJson {
+
+    public static String[] getPoolJson() {
+        return new String[] {
```

Comment:
nit: this would be a lot nice if it was in single line instead of each param in separate line.